ReleaseProcessAutomation/Jira/ServiceFacadeImplementations/JiraProjectVersionService.cs

```diff
@@ -21,22 +21,30 @@
 using System.Net;
 using ReleaseProcessAutomation.Jira.ServiceFacadeInterfaces;
 using ReleaseProcessAutomation.Jira.Utility;
+using ReleaseProcessAutomation.ReadInput;
 using ReleaseProcessAutomation.SemanticVersioning;
 using RestSharp;
+using Spectre.Console;
 
 namespace ReleaseProcessAutomation.Jira.ServiceFacadeImplementations;
 
 public class JiraProjectVersionService : IJiraProjectVersionService
 {
+  private readonly IInputReader _inputReader;
+  private readonly IAnsiConsole _console;
```

Comment:
I kind of dislike that we have to pull in the input/output into this service. Ideally the services just provide the functionality and the steps are the ones that do the interaction and invoke the right functionality. If we start crossing lines it gets messy quickly.

Maybe we can extract this into a step to keep them separated? What do you think?